index.js

```diff
@@ -1,13 +1,13 @@
 'use strict'
 
-var window = require('global/window')
-
-var OfflineContext = window.OfflineAudioContext || window.webkitOfflineAudioContext
-var Context = window.AudioContext || window.webkitAudioContext
-
 var cache = {}
 
 module.exports = function getContext (options) {
+	if (!window) return null
+	
+	var OfflineContext = window.OfflineAudioContext || window.webkitOfflineAudioContext
+	var Context = window.AudioContext || window.webkitAudioContext
```

Comment:
Obtaining context class each call might be not the best idea performance-wise, although having out-of-scope reference as it is done now is possibly no-good too.